src/features/todos.ts

```diff
@@ -1,9 +1,28 @@
 import { Todo } from '../types/Todo';
 
-export const actions = {};
+type SetTodosAction = {
+  type: 'currentTodos/SET';
+  payload: Todo[];
+};
+
+const setTodos = (todos: Todo[]): SetTodosAction => ({
+  type: 'currentTodos/SET',
+  payload: todos,
+});
+
+export const actions = { setTodos };
```

Comment:
Function declarations should have a blank line before and after them. Consider adding a blank line after the setTodos function declaration.

Reply:
I'm not quite sure I understand what you mean. Function setTodos is actually function expression and there are blank lines before and after it. If you mean to add blank line right after the line where function where linked to the veriable so I did it (pls see my fixes)